docs/docs/internal/governance--create-proposal.md

```diff
@@ -0,0 +1,86 @@
+3. PICA available for proposal submission
+4. PICA available for Pre- Image submission
+
+
+### Process
+By following the upcoming steps, you will create a governance proposal on Dali/Picasso/Composable.
+
+#### 1. Submitting a Pre- Image
+To submit a Pre-Image for a proposal, go to the democracy page of the PolkadotJS web interface. Then click on the "Submit preimage" button.
+Select the desired proposals extrinsic an outcome, note down the preimage hash, and press "Submit preimage".
+
+![](./partake-in-governance--images/1-governance--preimage-submission.png)
+
+
+#### 2. Submitting a proposal
+To submit a proposal, go to the democracy page of the PolkadotJS web interface. Then click on the "Submit proposal" button.
+Enter your "preimage hash" in the appearing text box, and select the amount of balance you want to lock with your proposal.
+
+
+#### 3. Getting proposal into referenda
+By now you will see your proposals in the "proposals" category on the democracy tab of the PolkadotJS web interface.
+Here, other people can endorse your proposal, and the proposal with most endorsements will get into referenda at the start of the next launch period.
+The launch period depends on the corresponding chain in use.
+This is defined in the runtime configuration by setting the following parameter for the `Democracy` pallet.
+```rust
+pub const LaunchPeriod: BlockNumber = HOURS;
+```
+
+| Chain | Launch Period |  
+| ----------- | ----------- |  
+| Dali | 1 Hour |  
+| Picasso | 5 Days |
+| Composable | 5 Days |
+
+To endorse a proposal, select the "Endorse" button next to a proposal on the PolkadotJS web interface.
+
+![](./partake-in-governance--images/3-governance--endorse.png)
+
+
+#### 4. Voting
+Now that your proposal is in referenda. People can vote on the outcome of the proposal.
+Voters can increase their voting power by locking their tokens up for up to a time of their wish, dependent on the corresponding chain in use.
+For the vote to be successful, it will need a `50% + 1` approval majority of all votes.
+The voting period length can be defined by setting the following parameter for the `Democracy` pallet.
+```rust
+pub const LaunchPeriod: BlockNumber = HOURS;
+```
+
+| Chain | Voting Period |  
+| ----------- | ----------- |  
+| Dali | 5 Days |  
+| Picasso | 5 Days |
+| Composable | 5 Days |
+
+![Voting](./partake-in-governance--images/4-1-governance--referenda.png)
+![A](./partake-in-governance--images/4-2-governance--voting.png)
+
+
+#### 5. Enactment of proposal
+After the voting period has ended, and the turnout has been successful. There will be a delay until the proposal will get enacted.
+This time depends on the corresponding chain in use.
+This is defined in the runtime configuration by setting the following parameter for the `Democracy` pallet.
+```rust
+pub const EnactmentPeriod: BlockNumber = HOURS;
+```
+
+| Chain | Enactment Period |  
+| ----------- | ----------- |  
+| Dali | 1 Hour |  
+| Picasso | 2 Days |
```

Comment:
Can this be skipped when fast tracking?

Reply:
Yes, as mentioned in the how to fast track guide section 5. once the minimum voting threshold has been reached, any technical committee member can close and execute the proposal.

Reply:
So closing executes the proposal and it's confirmed?